cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -29,6 +29,9 @@
     {
       "name": "HelmPackagesService"
     },
+    {
+      "name": "HelmRepositoriesService"
+    },
```

Comment:
No action required in this PR, but just mentioning it for all us to be aware: the `openapi.yaml` file (which is the doc used for the interactive REST API portal) will get out of sync.

Also, but unrelated: I don't know why my generator is different (meaning worse) than yours haha.